### ISSUE TITLE
learning rate, and weight decay.

### ISSUE THREAD

**Issue by presmann** [rainmaker22/SMART#28]
Thanks for sharing this contribution with the community.
I have a question regarding the learning rate and weight decay.
1. In the paper you've mentioned that you use `weight_decay=0.1`; however I don't see that in the code, and you call `ADAMW` which by default uses weight_decay=0.01.
2. The learning rate in the paper is 2e-4 (+cosine annealing); while in the supplied config file the parameter is set to 5e-4. 

Can you please advise on those settings? Which one led to your excellent WOSC results?

Thanks again,
Noam

**Comment by rainmaker22** [comment]
In practice, we used weight_decay=0.01 for our experiments, with an initial learning rate of 5e-4 and cosine annealing for learning rate decay.

**Comment by presmann** [comment]
Thank you for sharing that. 
Could you also share how many epochs your training took?

**Comment by rainmaker22** [comment]
about 32 epochs

**Comment by presmann** [comment]
Thank you for sharing that 
I've tried running the training with 32 GPUs and 32 epochs, with `lr = 5e-4`. The final validation accuracy (after 32 epocsh) is `val_cls_acc = 0.589` (please see the Tensorboard output below. For reference after the first epoch the validation accuracy is 0.578). May I ask if this is the accuracy you got after 32 epochs?

Thanks again, 

![Image](https://github.com/user-attachments/assets/19cca382-63d5-4d41-bb57-fc6d5b3f23cb)

![Image](https://github.com/user-attachments/assets/9956164a-2b05-4ddd-bc0e-b6c63c483cb1)

**Comment by rainmaker22** [comment]
looks reasonable
